airflow/providers/google/cloud/transfers/azure_fileshare_to_gcs.py

```diff
@@ -0,0 +1,210 @@
+
+    :param share_name: The Azure FileShare share where to find the objects. (templated)
+    :type share_name: str
+    :param directory_name: (Optional) Path to Azure FileShare directory which content is to be transferred.
+        Defaults to root directory (templated)
+    :type directory_name: str
+    :param prefix: Prefix string which filters objects whose name begin with
+        such prefix. (templated)
+    :type prefix: str
+    :param wasb_conn_id: The source WASB connection
+    :type wasb_conn_id: str
+    :param gcp_conn_id: (Optional) The connection ID used to connect to Google Cloud.
+    :type gcp_conn_id: str
+    :param dest_gcs_conn_id: (Deprecated) The connection ID used to connect to Google Cloud.
+        This parameter has been deprecated. You should pass the gcp_conn_id parameter instead.
+    :type dest_gcs_conn_id: str
+    :param dest_gcs: The destination Google Cloud Storage bucket and prefix
+        where you want to store the files. (templated)
+    :type dest_gcs: str
+    :param delegate_to: Google account to impersonate using domain-wide delegation of authority,
+        if any. For this to work, the service account making the request must have
+        domain-wide delegation enabled.
+    :type delegate_to: str
+    :param replace: Whether you want to replace existing destination files
+        or not.
+    :type replace: bool
+    :param gzip: Option to compress file for upload
+    :type gzip: bool
+    :param google_impersonation_chain: Optional Google service account to impersonate using
+        short-term credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+    :type google_impersonation_chain: Union[str, Sequence[str]]
+
+
+    **Example**:
+
+    .. code-block:: python
+
+       azure_fileshare_to_gcs_op = AzureFileShareToGCSOperator(
+            task_id='azure_fileshare_to_gcs_example',
+            share_name='my-azure-share',
+            directory_name='mydir/anotherdir',
+            prefix='data/customers-201804',
+            wasb_conn_id='azure_fileshare_default',
+            gcp_conn_id='google_cloud_default',
+            dest_gcs='gs://my.gcs.bucket/some/customers/',
+            replace=False,
+            gzip=True,
+            google_impersonation_chain=['account1', 'account2', 'account3'],
+            dag=my-dag
+       )
+
+    Note that ``share_name``, ``directory_name``, ``prefix``, ``delimiter`` and ``dest_gcs`` are
+    templated, so you can use variables in them if you wish.
+    """
```

Comment:
I would be in favor of removing this and using example in howto docs

Reply:
example is already used in howto docs so I just need to remove it from here